zql/docs/search-syntax/README.md

```diff
@@ -80,7 +80,7 @@ The search result can be narrowed to include only events that contain certain va
 
 ### Bare Word
 
-The simplest form of such a search is a "bare" word (not wrapped in quotes), which will match against any field value that contains the word as a substring.
+The simplest form of such a search is a "bare" word (not wrapped in quotes), which will match against any field that contains the word, whether it's an exact match to the data type and value of a field or the word appears as a substring in a field.
```

Comment:
@henridf: I made this change after seeing your PR review comment. Please give it a look.

Reply:
👍